### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by Mykola15**

### CODE REVIEW

**Review by KappasTV — APPROVED** [review]

File: public/mockups/doneTasksByUser.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  {
+    "id": 5,
+    "name": "Task1"
+  },
+  {
+    "id": 6,
+    "name": "Task2"
+  }
+]
```
Comment (wogy14) [inline public/mockups/doneTasksByUser.json]: Unused mockup.

File: package.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@loadable/component": "^5.12.0",
+    "@material-ui/core": "^4.9.9",
+    "@material-ui/icons": "^4.9.1",
```
Comment (wogy14) [inline package.json]: Unused modules

File: public/mockups/user.json [hunk 3]
```diff
@@ -1,13 +1,6 @@
...
-  "achievements": [{
-    "id": 1,
-    "name": "Hero"
-  }, {
-    "id": 2,
-    "name": "Helper"
-  }],
-  "totalAmount": 10
+  "photo": "http://localhost:3000/img/avatar.png"
 }
```
Comment (wogy14) [inline public/mockups/user.json]: Unused mockup.

File: src/components/ManageTask/ManageTask.tsx [hunk 4]
```diff
@@ -1,7 +1,8 @@
 import Modal from "react-bootstrap/Modal";
 import Button from "react-bootstrap/esm/Button";
 import React, {Component, FormEvent} from "react";
-import avatar from '../Task/task.svg';
+import { Image} from '@material-ui/icons';
```
Comment (wogy14) [inline src/components/ManageTask/ManageTask.tsx]: We use only FontAwesome. Please investigate if this module is used on this file.

**Pull request merged**